tests/test_external_tests.py

```diff
@@ -112,18 +143,33 @@ def test_that_the_libtiff_info_test_correctly_interprets_exit_code_0_and_1(
         assert test_status == TestStatus.FAIL
 
 
-def test_that_the_libtiff_info_test_command_is_produced(test_targets):
+def test_that_the_bioformats_info_test_command_is_produced(test_targets):
     target = test_targets["tiff"]
-    test = tests.LibTiffInfoTest(target)
+    test = tests.BioFormatsInfoTest(target)
     process = test.generate_process()
-    assert "tiffinfo" in process.command
+    assert "showinf" in process.command
 
 
-def test_that_the_bioformats_info_test_command_is_produced(test_targets):
-    target = test_targets["tiff"]
+@docker_enabled_test
+def test_that_the_bioformats_info_test_exit_code_is_0_when_it_should_be(test_files):
```

Comment:
Food for thought (feel free to take it or leave it): 
You did a great job splitting the tests up into the various python scripts - One area that could be improved is to have 1 more level of organization within the script by grouping like tests together. For example:

```
class TestBioformats:
    def test_....
    def test_....

class TestTiffDateTimeTest:
    def test_...
```

Further reading: https://docs.pytest.org/en/7.1.x/getting-started.html#group-multiple-tests-in-a-class

Reply:
I did consider this and have [used test classes](https://github.com/Sage-Bionetworks/agora-data-tools/blob/32e6f941c114934414baa5ffcf6a34075bca7394/tests/test_process.py#L13) in the past